scripts/two_line_bash.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+#
+#
+# This was casuing issues in tmux and always showed an "catkin_mil_ws", I'm sure that was intended
```

Comment:
This shows the currently sourced catkin workspace. It is intended to compliment wsmux since we are using different workspaces as opposed to user accounts on vehicles going forward.

I did not realize so few people had only a single workspace until @DSsoto brought it up, so I will be adding logic to only show this when there are multiple workspaces in `~/`. As for the tmux issues, I will look into that once I have made my changes.

I do not recommend altering this file right now or there will be many a merge conflict with my fork. If you insist, though, do it on `master` as these changes were not a part of ***The Great Merge***.